src/set_theory/cardinal_divisibility.lean

```diff
@@ -0,0 +1,141 @@
+/-
+Copyright (c) 2022 Eric Rodriguez. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Rodriguez
+-/
+
+import set_theory.cardinal_ordinal
+import algebra.is_prime_pow
+
+/-!
+# Cardinal Divisibility
+
+We show basic results about divisibility in the cardinal numbers. This relation can be characterised
+in the following simple way: if `a` and `b` are both less than `ω`, then `a ∣ b` iff they are
+divisible as natural numbers. If `b` is greater than `ω`, then `a ∣ b` iff `a ≤ b`. This furthermore
+shows that all infinite cardinals are prime; recall that `a * b = max a b` if `ω ≤ a * b`; therefore
+`a ∣ b * c = a ∣ max b c` and therefore clearly either `a ∣ b` or `a ∣ c`. Note furthermore that
+no infinite cardinal is irreducible (`cardinal.not_irreducible_of_omega_le`), showing that the
+cardinal numbers do not form a `comm_cancel_monoid_with_zero`.
```

Comment:
General question: would the API benefit from having `cardinal.infinite` for use with dot notation? It would just be defined to be `ω ≤ a`.

Reply:
I'm not really a fan, since it obscures a quite simple definition.

Reply:
I feel like the `le` dot notation works pretty well with it already, too.